glacierscript.py

```diff
@@ -40,6 +40,8 @@
 
 SATOSHI_PLACES = Decimal("0.00000001")
 
+VERBOSE_MODE = 0
```

Comment:
In python, all-caps is a convention for global constants (see [Google's python style guide](https://google.github.io/styleguide/pyguide.html). In this case, this is not constant, as a cmdline option could change it.